api/src/main/java/com/epam/pipeline/security/acl/AclExpressions.java

```diff
@@ -132,6 +132,12 @@
     public static final String OR_HAS_ASSIGNED_USER_OR_ROLE =
             " OR @grantPermissionManager.hasCloudProfilePermissions(#profileId)";
 
+    public static final String ADMIN_OR_HAS_READ_ACCESS_ON_ENTITIES_FROM_LIST =
+        ADMIN_ONLY + OR + "hasPermission(filterObject, 'READ')";
+
+    public static final String ADMIN_OR_HAS_READ_ACCESS_ON_ENTITY =
+        ADMIN_ONLY + OR + "hasPermission(filterObject, 'READ')";
```

Comment:
In case of `@PostAuthorize` annotation is shall be `hasPermission(returnObject, 'READ')`
Could you please fix it and rename constant to `ADMIN_OR_HAS_READ_ACCESS_ON_RETURN_OBJECT`?

Reply:
Fixed